control-plane/agents/common/src/errors.rs

```diff
@@ -121,6 +125,14 @@ pub enum SvcError {
     WatchAlreadyExists {},
     #[snafu(display("Conflicts with existing operation - please retry"))]
     Conflict {},
+    #[snafu(display("Pending deletion - please retry"))]
+    Deleting {},
+    #[snafu(display(
```

Comment:
Would it be useful to show both sets of parameters so that it is obvious what differs? Or maybe we could show the parameters that differ?

Reply:
Yeah that'd be useful indeed :+1:

Reply:
Using the Debug trait to generate the string for now, might want to think about using the Serialize or Display instead...